plugins/intrinsics/src/main/java/org/qbicc/plugin/intrinsics/core/CoreIntrinsics.java

```diff
@@ -87,7 +85,11 @@ public static void register(CompilationContext ctxt) {
         registerJavaLangSystemIntrinsics(ctxt);
         registerJavaLangStackTraceElementInstrinsics(ctxt);
         registerJavaLangThreadIntrinsics(ctxt);
-        registerJavaLangThrowableIntrinsics(ctxt);
+        if (ctxt.getPlatform().getCpu() != Cpu.WASM32) {
+            registerEmptyJavaLangThrowableIntrinsics(ctxt);
+        } else {
+            registerJavaLangThrowableIntrinsics(ctxt);
+        }
```

Comment:
whoopsie I think this slipped the both of us @dmlloyd this is supposed to read '=='. I have no idea how come this has worked so far... (but probably I have made some mistake while cherry-picking from some integration branch I have locally)